torch_xla/csrc/layout_manager.cpp

```diff
@@ -185,9 +185,7 @@ xla::Shape MakeArrayShapeFromDimensions(
     return MakeShapeWithLayout(type, dimensions, dynamic_dimensions,
                                *layout_ptr);
   }
-
-  bool tpu_layout_env = runtime::sys_util::GetEnvBool("XLA_TPU_LAYOUT", true);
-  if (tpu_layout_env && dimensions.size() > 1 && CheckTpuDevice(hw_type)) {
```

Comment:
I don't understand why `tpu_layout_env` should be deprecated and how is it related to this pr? Are you saying we always use `XLA_TPU_LAYOUT ` ? Does that has a implication on performance on GPU for example?

Reply:
well I guess it is default to true anyway.. but still I am not sure why it is deleted in this pr.

Reply:
I added that to partially solve this problem. Now I found the completed solution, and therefore the old solution is no longer needed.